scripts/test-local.sh

```diff
@@ -2,43 +2,39 @@
 set -e
 set -x
 
- #
- # Copyright (c) 2023 Project CHIP Authors
- #
- # Licensed under the Apache License, Version 2.0 (the "License");
- # you may not use this file except in compliance with the License.
- # You may obtain a copy of the License at
- #
- # http://www.apache.org/licenses/LICENSE-2.0
- #
- # Unless required by applicable law or agreed to in writing, software
- # distributed under the License is distributed on an "AS IS" BASIS,
- # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- # See the License for the specific language governing permissions and
- # limitations under the License.
-
-
+#
+# Copyright (c) 2023 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
 
 #!/bin/sh
-for arg in "$@"
-do
+for arg in "$@"; do
     case $arg in
-        --run-platform-dependant)
+    --run-platform-dependant)
         RUN_ALL_TESTS=1
         shift # Remove --run-all from processing
         ;;
-        *)
+    *)
```

Comment:
Looks like there's been some autoformating of this file.

What editor and settings did you use? I wonder if we should revert all he non-essential edits here. 

But if you saved it from VSCode with our standard tooling, we can keep the changes.

Reply:
Yeah, I used VSCode and it did that automatically